charts/ratify/templates/configmap.yaml

```diff
@@ -7,7 +7,9 @@ data:
     {
       "executor": {
         "verificationRequestTimeout": {{ .Values.provider.timeout.validationTimeoutSeconds | int | mul 1000 | add -100 }},
-        "mutationRequestTimeout": {{ .Values.provider.timeout.mutationTimeoutSeconds | int | mul 1000 | add -50 }}
+        "mutationRequestTimeout": {{ .Values.provider.timeout.mutationTimeoutSeconds | int | mul 1000 | add -50 }},
+        "useRegoPolicy": {{ .Values.policyProvider.useRegoPolicy }},
```

Comment:
curious why we need to have a rego policy flag in the executor section? Wouldn't that be the job of the policy provider?

Reply:
synced in pr meeting, moved it as feature flag.